### PR TITLE
added reset button + functionality

### PR DESCRIPTION
**Pull request by janessatran**

### CODE REVIEW

File: timer.js [hunk 1]
```diff
@@ -36,18 +36,19 @@ function play() {
     }
     else
         startTimer();
+    timerRunning = True;
```
Comment (botanical) [inline timer.js:40]: maybe add four spaces so it aligns with `startTimer()`? 🙂

File: timer.js [hunk 2]
```diff
@@ -36,18 +36,19 @@ function play() {
...
-function stop() {
-
-}
-
 function reset() {
-    
+    console.log('resetting timer!')
+    minutes = 25;
+    seconds = 0;
+    setTime();;
```
Comment (botanical) [inline timer.js]: does it need two `;;`?

File: timerSpec.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+describe(function(){
+    var form;
+  
+    beforeEach(function(){
+      form = $("<button class='button play' id='play'></button> ");
+      $(document.body).append(form);
+    });
+  
+    it('should call the play() function when clicked', function(){
+        var spyEvent = spyOnEvent($('#play'), 'click');
```
Comment (botanical) [inline timerSpec.js:10]: nice!!! 😁

File: timerSpec.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+describe(function(){
```
Comment (botanical) [inline timerSpec.js:1]: I think test should be named `timer.test.js` and it should go in a `__tests__` directory

File: timer.js (hunk 1 above)
Comment (janessatran) [reply timer.js:40]: ohh good catch

File: timer.js (hunk 2 above)
Comment (janessatran) [reply timer.js]: no it doesn't, another good catch!

**Pull request merged**